<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
	<class>Settings</class>
	<widget class="QDialog" name="Settings">
		<property name="windowModality">
			<enum>Qt::WindowModal</enum>
		</property>
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>470</width>
				<height>187</height>
			</rect>
		</property>
		<property name="sizePolicy">
			<sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
				<horstretch>0</horstretch>
				<verstretch>0</verstretch>
			</sizepolicy>
		</property>
		<property name="windowTitle">
			<string>SongReader.Settings.DialogTitle</string>
		</property>
		<layout class="QGridLayout" name="gridLayout">
			<item row="0" column="0">
				<widget class="QGroupBox" name="portGroupBox">
					<property name="sizePolicy">
						<sizepolicy hsizetype="Preferred" vsizetype="Fixed">
							<horstretch>0</horstretch>
							<verstretch>0</verstretch>
						</sizepolicy>
					</property>
					<property name="title">
						<string>SongReader.Settings.WebServer.Active</string>
					</property>
					<property name="checkable">
						<bool>true</bool>
					</property>
					<property name="checked">
						<bool>false</bool>
					</property>
					<layout class="QFormLayout" name="formLayout_3">
						<item row="0" column="0" colspan="2">
							<layout class="QFormLayout" name="formLayout_2">
								<item row="0" column="0">
									<widget class="QLabel" name="mainOutputNameLabel">
										<property name="minimumSize">
											<size>
												<width>200</width>
												<height>0</height>
											</size>
										</property>
										<property name="text">
											<string>SongReader.Settings.Port</string>
										</property>
									</widget>
								</item>
								<item row="0" column="1">
									<widget class="QLineEdit" name="portNumber"/>
								</item>
							</layout>
						</item>
					</layout>
				</widget>
			</item>
			<item row="1" column="0">
				<widget class="QGroupBox" name="fileGroupBox">
					<property name="sizePolicy">
						<sizepolicy hsizetype="Preferred" vsizetype="Fixed">
							<horstretch>0</horstretch>
							<verstretch>0</verstretch>
						</sizepolicy>
					</property>
					<property name="minimumSize">
						<size>
							<width>0</width>
							<height>0</height>
						</size>
					</property>
					<property name="title">
						<string>SongReader.Settings.GroupBox.File</string>
					</property>
					<property name="alignment">
						<set>Qt::AlignJustify|Qt::AlignVCenter</set>
					</property>
					<property name="flat">
						<bool>false</bool>
					</property>
					<property name="checkable">
						<bool>true</bool>
					</property>
					<property name="checked">
						<bool>false</bool>
					</property>
					<layout class="QFormLayout" name="formLayout_5">
						<item row="0" column="0" colspan="2">
							<layout class="QFormLayout" name="formLayout_4">
								<item row="0" column="0">
									<widget class="QLabel" name="fileNameLabel">
										<property name="minimumSize">
											<size>
												<width>200</width>
												<height>0</height>
											</size>
										</property>
										<property name="maximumSize">
											<size>
												<width>16777215</width>
												<height>16777215</height>
											</size>
										</property>
										<property name="text">
											<string>SongReader.Settings.File</string>
										</property>
									</widget>
								</item>
								<item row="0" column="1">
									<widget class="QLineEdit" name="fileName"/>
								</item>
							</layout>
						</item>
					</layout>
				</widget>
			</item>
			<item row="2" column="0">
				<widget class="QDialogButtonBox" name="buttonBox">
					<property name="orientation">
						<enum>Qt::Horizontal</enum>
					</property>
					<property name="standardButtons">
						<set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
					</property>
				</widget>
			</item>
		</layout>
	</widget>
	<resources/>
	<connections>
		<connection>
			<sender>buttonBox</sender>
			<signal>accepted()</signal>
			<receiver>Settings</receiver>
			<slot>accept()</slot>
			<hints>
				<hint type="sourcelabel">
					<x>248</x>
					<y>254</y>
				</hint>
				<hint type="destinationlabel">
					<x>157</x>
					<y>274</y>
				</hint>
			</hints>
		</connection>
		<connection>
			<sender>buttonBox</sender>
			<signal>rejected()</signal>
			<receiver>Settings</receiver>
			<slot>reject()</slot>
			<hints>
				<hint type="sourcelabel">
					<x>316</x>
					<y>260</y>
				</hint>
				<hint type="destinationlabel">
					<x>286</x>
					<y>274</y>
				</hint>
			</hints>
		</connection>
	</connections>
</ui>